### PR TITLE
Base dashboard UI without data

### LINKED ISSUE #115: Dashboard - generic, no state
- [ ] generic dashboard that does not include the side menu
- [ ] no state for whether or not hacker was accepted/has RSVPed
- [ ] no functionality for edit and RSVP

Screen Shot 2020-11-21 at 4 12 41 PM
Screen Shot 2020-11-21 at 4 13 13 PM
Screen Shot 2020-11-21 at 4 13 19 PM

### PR DESCRIPTION
**Pull request by andykwan1124**
Closes #115. No firebase data yet, will make another PR when that is included.

### CODE REVIEW

File: src/theme/ThemeProvider.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ const nwTheme = {
       googleHover: '#fafafa',
       githubHover: '#545159',
     },
+    appDashboardCard: 'rgba(75, 65, 130, 0.2)',
```
Comment (ianmah) [inline src/theme/ThemeProvider.js:36]: where is this being used?

File: src/components/ApplicationDashboard.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+import { Button } from './Input/Button'
+import { SOCIAL_LINKS } from '../utility/Constants'
+import facebook from '../assets/icons/facebook.svg'
+import instagram from '../assets/icons/instagram.svg'
+import medium from '../assets/icons/medium.svg'
+import twitter from '../assets/icons/twitter.svg'
+import { ReactComponent as HandWave } from '../assets/hand-wave.svg'
+
+const Container = styled.div`
+  margin: 5em auto 10em;
```
Comment (ianmah) [inline src/components/ApplicationDashboard.js:13]:
```suggestion
  margin: 5em auto;
```

File: src/components/ApplicationDashboard.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+const AppStatusText = styled.p`
+  font-size: 1.25em;
+  font-weight: 700;
+  margin-top: 0;
+`
+
+const StatusBlurbText = styled.p`
+  color: ${p => p.theme.colors.primary};
+  font-size: 1.5em;
```
Comment (ianmah) [inline src/components/ApplicationDashboard.js:67]:
```suggestion
```

File: src/theme/ThemeProvider.js (hunk 1 above)
Comment (andykwan1124) [reply src/theme/ThemeProvider.js:36]: That's the background color of the card in the application page. Should I name it colors.applicationCard?

Comment (andykwan1124) [reply src/theme/ThemeProvider.js:36]: lmao seems like I forgot to change the hardcoded background-color to use the theme color

File: src/components/ApplicationDashboard.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+import { SOCIAL_LINKS } from '../utility/Constants'
+import facebook from '../assets/icons/facebook.svg'
+import instagram from '../assets/icons/instagram.svg'
+import medium from '../assets/icons/medium.svg'
+import twitter from '../assets/icons/twitter.svg'
+import { ReactComponent as HandWave } from '../assets/hand-wave.svg'
+
+const Container = styled.div`
+  margin: 5em auto;
+  width: 70%;
```
Comment (panjenny0) [inline src/components/ApplicationDashboard.js:14]: can we add a media query here and set width to 100% for mobile? the content looks too skinny and gets pushed down too much
Screen Shot 2020-12-01 at 1 11 05 PM
```suggestion
  width: 70%;
  ${maxWidthMediaQueries('mobile')} {
    margin: 0;
    width: 100%;
  }
```

File: src/components/ApplicationDashboard.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+const WelcomeHeader = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 1em;
+`
+
+const WelcomeMessage = styled(H1)`
+  text-align: center;
```
Comment (panjenny0) [inline src/components/ApplicationDashboard.js:25]:
```suggestion
  text-align: center;
  ${maxWidthMediaQueries('mobile')} {
  font-size: 1.5em;
  }
```

File: src/components/ApplicationDashboard.js [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+const StatusContainer = styled.div`
+  min-height: 350px;
+  padding: 2.5em 4.5em 2.5em 3em;
+  border-radius: 21px;
+  background-color: ${p => p.theme.colors.applicationCard};
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
```
Comment (panjenny0) [inline src/components/ApplicationDashboard.js:57]: Let's make `padding: 2em` for mobile
(what it looks like)
Screen Shot 2020-12-01 at 1 30 15 PM

File: src/components/ApplicationDashboard.js [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+
+const FooterContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: -24px;
+`
+
+const SocialMediaIcons = styled.img`
+  margin: 24px;
```
Comment (panjenny0) [inline src/components/ApplicationDashboard.js:80]: Screen Shot 2020-12-01 at 1 13 57 PM
These are pretty much vertical on mobile, let's shrink margin to 8px; and the size of the icons to 24px; in mobile

```suggestion
  margin: 24px;
  ${maxWidthMediaQueries('mobile')} {
width: 24px;
height: 24px;
  margin: 16px;
  }
```

File: src/components/ApplicationDashboard.js [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+  font-weight: 700;
+  line-height: 2em;
+  margin-top: 0.75em;
+`
+
+const FooterContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: -24px;
```
Comment (panjenny0) [inline src/components/ApplicationDashboard.js:76]: let's make this centered and take up two lines on mobile
```suggestion
  display: flex;
  justify-content: space-between;
  align-items: center;
  margin-left: -24px;
  ${maxWidthMediaQueries('mobile')} {
 display: block;
text-align: center;
  }
```

Screen Shot 2020-12-01 at 1 45 46 PM

File: src/components/ApplicationDashboard.js [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+`
+
+const SocialMediaIcons = styled.img`
+  margin: 24px;
+`
+
+const RSVPButton = styled(Button)`
+  width = 100px;
+  margin-right: 0;
```
Comment (panjenny0) [inline src/components/ApplicationDashboard.js:85]:
```suggestion
  margin-right: 0;
  ${maxWidthMediaQueries('mobile')} {
margin: 1em;
  }
```
to center the button in mobile (see one of my above screenshots)

**Review by panjenny0 — CHANGES_REQUESTED** [review]
Good job! Added some suggestions

**Review by panjenny0 — APPROVED** [review]
👍

**Pull request merged**